fbw-a380x/src/systems/systems-host/systems/tcas/lib/TcasConstants.ts

```diff
@@ -0,0 +1,849 @@
+// Copyright (c) 2021-2023 FlyByWire Simulations
+//
+// SPDX-License-Identifier: GPL-3.0
+
+/* eslint-disable camelcase */
+const REFRESH_RATE: number = 1000; // Time between refreshes in ms
+const TRACKING_MAX: number = 40; // # max contacts tracked - AMM 34-43-00:6a
+const DISPLAY_MAX: number = 8; // # max contacts tracked - usually configurable by PIN program
+const MEMORY_MAX: number = 200; // Max history before culling
+const MIN_VS: number = -6000;
+const MAX_VS: number = 6000;
+const INHIBIT_CLB_RA: number = 39000; // for all climb RA's
+const INHIBIT_INC_DES_RA_AGL: number = 1450; // for increase descent RA's
+const INHIBIT_ALL_DES_RA_AGL: number = 1200; // 1200 takeoff, 1000 approach
+const INHIBIT_ALL_RA: number = 1000; // 1100 in climb, 900 in descent
+const REALLY_BIG_NUMBER: number = 1000000;
+const INITIAL_DELAY: number = 5; // in seconds
+const FOLLOWUP_DELAY: number = 2.5; // in deconds
+const INITIAL_ACCEL: number = 8.04; // 0.25G in f/s^2
+const FOLLOWUP_ACCEL: number = 10.62; // 0.33G in f/s^2
+const TA_EXPIRATION_DELAY: number = 4; // in seconds
+const MIN_RA_DURATION: number = 5; // in seconds
+const VOL_BOOST: number = 1.25; // multiplier
+const CLOSURE_RATE_THRESH: number = -40; // in knots
+
+export enum TcasThreat {
+  THREAT = 0,
+  ALL = 1,
+  ABOVE = 2,
+  BELOW = 3,
+}
+
+export enum TcasDirection {
```

Comment:
```suggestion
export enum TcasAltitudeSelect {
```
The proper name for this input.